emcc.py

```diff
@@ -3571,6 +3574,8 @@ def consume_arg_file():
       settings.DISABLE_EXCEPTION_CATCHING = 1
       settings.DISABLE_EXCEPTION_THROWING = 1
       settings.WASM_EXCEPTIONS = 0
+    elif arg == '-mbulk-memory':
```

Comment:
Do we need to do anything to handle `-mno-bulk-memory`?

Reply:
Since the new (internal) BULK_MEMORY defaults to false I don't think so.

Reply:
What about handling e.g. `-mbulk-memory -mno-bulk-memory`?

Reply:
Fair enough :)